admin/deployments.go

```diff
@@ -52,25 +52,25 @@ func (s *Service) createDeployment(ctx context.Context, opts *createDeploymentOp
 	olapConfig := map[string]string{}
 	var embedCatalog bool
 	switch olapDriver {
-	case "duckdb":
+	case "duckdb-int-storage": // duckdb driver that disables storing table as view
 		if opts.ProdOLAPDSN != "" {
-			return nil, fmt.Errorf("passing a DSN is not allowed for driver 'duckdb'")
+			return nil, fmt.Errorf("passing a DSN is not allowed for driver 'duckdb-int-storage'")
 		}
 		if opts.ProdSlots == 0 {
-			return nil, fmt.Errorf("slot count can't be 0 for driver 'duckdb'")
+			return nil, fmt.Errorf("slot count can't be 0 for driver 'duckdb-int-storage'")
 		}
 
 		olapConfig["dsn"] = fmt.Sprintf("%s.db", path.Join(alloc.DataDir, instanceID))
 		olapConfig["cpu"] = strconv.Itoa(alloc.CPU)
 		olapConfig["memory_limit_gb"] = strconv.Itoa(alloc.MemoryGB)
 		olapConfig["storage_limit_bytes"] = strconv.FormatInt(alloc.StorageBytes, 10)
 		embedCatalog = false
-	case "duckdb-ext-storage": // duckdb driver having capability to store table as view
+	case "duckdb":
```

Comment:
There are still old deployments with `duckdb-ext-storage` in the DB, so I think we should support that case here as well (for resets)

Reply:
I was planning to change all of these to `duckdb` before release but yeah handling both is better.